### DISCUSSION TITLE
How many ports does a full node need to map?

### DISCUSSION THREAD

**Discussion by hedandan1989** [Chia-Network/chia-blockchain#2875] [General]
How many ports does a full node need to map?
I am behind a firewall, so I did 8444 NAT mapping. Now you can sync normally.
But do other ports need to be mapped? Are all people on the Internet only communicating through 8444?

**Comment by cccat6** [comment]
Actually, no need for opening 8444, and you can still get syncing. It just for the sharing and opening sprits. Open 8444 will be able to be a part of hosts on the Chia network and it is great.
8444 is the only port that you need to open for sure.

**Comment by dorkmo** [comment]
if you have seperate computers running as harvesters on the network, they communicate locally on different ports
